mysql-test/t/percona_heap_var.test

```diff
@@ -165,3 +165,44 @@ ALTER TABLE t1 ADD PRIMARY KEY (a, c);
 --error ER_TABLE_CANT_HANDLE_BLOB
 ALTER TABLE t1 ADD INDEX idx (c, a);
 DROP TABLE t1;
+
+create table `t1` (
```

Comment:
Please keep the SQL keyword case consistent throughout the testcase

Reply:
The file is already inconsistent, upper case characters are only used by the tests for bug173483, added recently - and I followed the way the older parts looks like, as I originally added this change at the gca point.

Should I change the first or the latter part of the test?

Reply:
Oh, my fault then. I'd prefer uppercase then